CMakeLists.txt

```diff
@@ -68,6 +68,30 @@ install(FILES
     ${CMAKE_CURRENT_BINARY_DIR}/${PACKAGE_NAME}-config-version.cmake
   DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/${PACKAGE_NAME})
 
+# Create pkg-config file
+include(build/JoinPaths.cmake)
+# from: https://github.com/jtojnar/cmake-snips#concatenating-paths-when-building-pkg-config-files
+join_paths(DIRECTXMATH_INCLUDEDIR_FOR_PKG_CONFIG "\${prefix}" "${CMAKE_INSTALL_INCLUDEDIR}")
+join_paths(DIRECTXMATH_LIBDIR_FOR_PKG_CONFIG "\${prefix}"     "${CMAKE_INSTALL_LIBDIR}")
+
+set(DIRECTXMATH_DEP_L "")
+if(directxmath_FOUND)
```

Comment:
This library doesn't depend on DirectX-Headers. You should remove tehse three lines I believe. Basically you can just delete all the lines for ``DIRECTXMATH_DEP_L`` and set `DIRECTXMATH_DEP` to blank I believe.